gensrc/thrift/Exprs.thrift

```diff
@@ -222,6 +222,7 @@ struct TExprNode {
   // Used for SubfieldExpr
   30: optional list<string> used_subfield_names;
   31: optional TBinaryLiteral binary_literal;
+  32: optional bool copy_flag;
```

Comment:
add comment this flag only used for subfield expr

Reply:
will do it